Example/CollectionViewToolsExample/Main/Factories/ContentViewSectionItemsFactory.swift

```diff
@@ -8,126 +8,143 @@ import CollectionViewTools
-            let width = collectionView.bounds.width
-            let aspectRatio = state.imageContent.image.size.width / state.imageContent.image.size.height
-            return CGSize(width: width, height: width / aspectRatio)
-        }
-        return factory
-    }()
-
-    // MARK: - TextContent
-    private lazy var textCellItemFactory: CellItemFactory = {
-        let factory: AssociatedCellItemFactory<TextViewState, TextCollectionViewCell> = makeFactory(id: "text")
-        let cellConfigurationHandler = factory.cellConfigurationHandler
-        factory.cellConfigurationHandler = { cell, cellItem in
-            cell.titleLabel.text = cellItem.object.textContent.text
-            cellConfigurationHandler?(cell, cellItem)
-            return
-        }
-        factory.sizeConfigurationHandler = { data, collectionView, sectionItem in
-            CGSize(width: collectionView.bounds.width, height: 60)
-        }
-        return factory
-    }()
-
-    // MARK: - Content
-    private lazy var cellItemFactory: CellItemFactory = {
-        imageCellItemFactory.factory(byJoining: textCellItemFactory)
-    }()
-
-    // MARK: - Description
-    private lazy var descriptionCellItemFactory: CellItemFactory = {
-        let factory = AssociatedCellItemFactory<ContentViewState, TextCollectionViewCell>()
-        factory.cellItemConfigurationHandler = { index, cellItem in
-            cellItem.itemDidSelectHandler = { _ in
-                cellItem.object.isExpanded.toggle()
-                self.output?.reloadCollectionView()
-            }
-        }
-        factory.cellConfigurationHandler = { cell, cellItem in
-            cell.titleLabel.text = cellItem.object.content.description
-        }
-        factory.sizeConfigurationHandler = { data, collectionView, sectionItem in
-            CGSize(width: collectionView.bounds.width, height: 60)
-        }
-        return factory
-    }()
+       // MARK: - Factories
+
+       // MARK: - ImageContent
+       private(set) lazy var imageCellItemFactory: CellItemFactory = {
+           let factory: AssociatedCellItemFactory<ImageViewState, ImageCollectionViewCell> = makeFactory(id: "image")
+           let cellConfigurationHandler = factory.cellConfigurationHandler
+           factory.cellConfigurationHandler = { cell, cellItem in
+               cell.imageView.image = cellItem.object.imageContent.image
+               cell.removeActionHandler = { [weak self] in
+                   self?.removeEventTriggered(state: cellItem.object)
+               }
+               cellConfigurationHandler?(cell, cellItem)
+           }
+           factory.sizeConfigurationHandler = { state, collectionView, sectionItem in
+               let width = collectionView.bounds.width
+               let aspectRatio = state.imageContent.image.size.width / state.imageContent.image.size.height
+               return CGSize(width: width, height: width / aspectRatio)
+           }
+           return factory
```

Comment:
```suggestion
           let factory: AssociatedCellItemFactory<ImageViewState, ImageCollectionViewCell> = makeFactory(id: "image")
           let cellConfigurationHandler = factory.cellConfigurationHandler
           
           factory.cellConfigurationHandler = { cell, cellItem in
               cell.imageView.image = cellItem.object.imageContent.image
               cell.removeActionHandler = { [weak self] in
                   self?.removeEventTriggered(state: cellItem.object)
               }
               cellConfigurationHandler?(cell, cellItem)
           }
           
           factory.sizeConfigurationHandler = { state, collectionView, sectionItem in
               let width = collectionView.bounds.width
               let aspectRatio = state.imageContent.image.size.width / state.imageContent.image.size.height
               return CGSize(width: width, height: width / aspectRatio)
           }
           
           return factory
```
Maybe add extra space between block ?